crates/utils/re_mcap/README.md

```diff
@@ -0,0 +1,10 @@
+# re_mcap
+
+Part of the [`rerun`](https://github.com/rerun-io/rerun) family of crates.
+
+[![Latest version](https://img.shields.io/crates/v/re_mcap)](https://crates.io/crates/re_mcap?speculative-link)
+[![Documentation](https://docs.rs/re_mcap/badge.svg)](https://docs.rs/re_mcap?speculative-link)
+![MIT](https://img.shields.io/badge/license-MIT-blue.svg)
+![Apache](https://img.shields.io/badge/license-Apache-blue.svg)
+
+Deserialization helpers for MCAP files.
```

Comment:
A better description would be "Convert MCAP into Rerun-compatible data" or something